src/nyc_trees/js/src/surveyPage.js

```diff
@@ -58,12 +61,11 @@ var tileLayer = mapModule.addTileLayer(blockfaceMap),
             $(dom.selectSide).addClass('hidden');
             $(dom.btnNextGroup).addClass('hidden');
 
-            mapUtil.zoomToBlockface(blockfaceMap, gridData.id);
+            mapUtil.zoomToBlockface(blockfaceMap, blockfaceId);
             return true;
         }
     });
 
-blockfaceMap.addLayer(grid);
```

Comment:
Looks like there is now a `addGridLayer` method on the map module that is called earlier on.